api/test/integration/test_agentsPUT_endpoints.tavern.yaml

```diff
@@ -296,6 +296,88 @@ stages:
       json:
         code: 1114
 
+---
+test_name: PUT /groups/{group_id}/restart
+
+stages:
+
+    # PUT /groups/group1/restart (This group has no agents assigned)
+  - name: Restart all agents from group1
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/{version:s}/groups/group1/restart"
+      method: PUT
+      headers:
+        Authorization: "Bearer {test_login_token}"
+        content-type: "application/xml"
+    response:
+      status_code: 400
+      json:
```

Comment:
I would remove all checks except code 1755 to avoid excess compromise with the messages.